common/position/position-en.html

```diff
@@ -0,0 +1,155 @@
+---
+{
+	"title": "GCWeb Positioning example",
+	"language": "en",
+	"altLangPage": "position-fr.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" }
+	],
+	"dateModified": "2023-06-19"
+}
+---
+<div class="wb-prettify all-pre"></div>
+
+<h2> Relative Positioning </h2>
+<p class="position-relative">This is a paragraph relatively positioned.</p>
+<h3> code </h3>
+<pre><code> &lt;p class="position-relative"&gt;This is a paragraph relatively positioned.&lt;/p&gt;
+</code></pre>
+
+<h2> Absolute positioning </h2>
+<h3><code> pstn-bttm-xs pstn-rght-xs </code></h3>
```

Comment:
Just be more descriptive instead of only having the technical name

```suggestion
<h3>Always bottom, always right (<code> pstn-bttm-xs pstn-rght-xs </code>)</h3>
```